### PR TITLE
Added AllVariables flag to HandlerVariablesAttribute and null initial…

### PR DESCRIPTION
**Pull request by ionut-openminds**
…ization of Variables list based on the flag

### CODE REVIEW

File: src/Camunda.Worker/Execution/HandlerMetadata.cs [hunk 1]
```diff
@@ -15,13 +15,14 @@ public HandlerMetadata(IReadOnlyList<string> topicNames, int lockDuration = Cons
         public int LockDuration { get; }
 
         public bool LocalVariables { get; set; }
+        public bool AllVariables { get; set; }
```
[the remark this answers is not in the document]
Comment (ionut-openminds) [reply src/Camunda.Worker/Execution/HandlerMetadata.cs:18]: Yes, you are right.

File: src/Camunda.Worker/HandlerVariablesAttribute.cs [hunk 2]
```diff
@@ -14,5 +14,7 @@ public HandlerVariablesAttribute(params string[] variables)
         public IReadOnlyList<string> Variables { get; }
 
         public bool LocalVariables { get; set; }
+
+        public bool AllVariables { get; set; }
```
[the remark this answers is not in the document]
Comment (ionut-openminds) [reply src/Camunda.Worker/HandlerVariablesAttribute.cs:18]: Sure

File: src/Camunda.Worker/CamundaWorkerBuilderExtensions.cs [hunk 3]
```diff
@@ -36,7 +36,7 @@ private static HandlerMetadata CollectMetadataFromAttributes(Type handlerType)
             return new HandlerMetadata(topicsAttribute.TopicNames, topicsAttribute.LockDuration)
             {
                 LocalVariables = variablesAttribute?.LocalVariables ?? false,
-                Variables = variablesAttribute?.Variables,
+                Variables =  variablesAttribute?.AllVariables ?? false ? null : variablesAttribute?.Variables,
```
Comment (AMalininHere) [inline src/Camunda.Worker/CamundaWorkerBuilderExtensions.cs:39]: Could you add the test for this behavior

Comment (ionut-openminds) [reply src/Camunda.Worker/CamundaWorkerBuilderExtensions.cs:39]: Sure

**Comment by AMalininHere** [comment]
Thank you for your contribution! I will prepare a new release later

**Comment by AMalininHere** [comment]
I just created a release with a new version: https://github.com/AMalininHere/camunda-worker-dotnet/releases/tag/0.13.4

**Pull request merged**